tools-local/tasks/GetBuildableFrameworks.cs

```diff
@@ -31,21 +32,20 @@ public override bool Execute()
             List<string> args = new List<string>();
             foreach (var projectJsonPath in ProjectJsonPaths)
             {
-                using (TextReader projectFileReader = File.OpenText(projectJsonPath.ItemSpec))
+                string dir = Path.GetDirectoryName(projectJsonPath.ItemSpec);
```

Comment:
This all falls out of the hacky way we are building the stuff under src/managed. If those projects were properly hooked into the build system (perhaps with a CoreFX-like configuration system), then we wouldn't need to be doing this craziness.